modules/private-clusters-default.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-private.adoc
+// * installing/installing_gcp/installing-gcp-private.adoc
+
+[id="private-clusters-default_{context}"]
+= Private clusters
+
+If your environment does not require an external internet connection, you can deploy a private {product-title} cluster that does not expose external endpoints. Private clusters are accessible from only an internal network and are not visible to the Internet.
```

Comment:
@jianlinliu, does this module address your concerns about context for why you need your own network and using the bastion host?

Reply:
LGTM.